plugins/git/git.plugin.zsh

```diff
@@ -32,6 +32,23 @@ function work_in_progress() {
     echo "WIP!!"
   fi
 }
+#change current branch old email with new eamil
+gmvemail() {
+  export OLD_GIT_EMAIL=$1
```

Comment:
using local does not work here, only export works well, I changed a name to avoid modifying env var by accident